aspnetcore/aspnetcore-2.0.md

```diff
@@ -74,11 +74,176 @@ For information about the status of planned documentation, see the [GitHub issue
 
 ## Logging update
 
-In ASP.NET 2.0, you configure logging in *Program.cs* instead of in your *Startup.cs* `Configure` method as in 1.x. Since configuration is also registered in *Program.cs*, *Startup.cs* is more focused on configuring services and the middleware pipeline. 
+Logging is now incorporated into the dependency injection (DI) system by default, and you add providers and configure filtering in the *Program.cs* file instead of in the *Startup.cs* file. Here's an example that shows how to add providers and specify that configuration is used to set up filtering:
```

Comment:
Now is ambiguous. How about:
In ASP.NET Core 2.0,  logging is now incorporated into the dependency injection (DI) system by default. Providers and filters are added to the *Program.cs* file instead of in the *Startup.cs* file. (need to split that sentence up somehow - too long for ESL and machine translator)